src/redist/redist.csproj

```diff
@@ -154,18 +154,6 @@
           Condition=" '$(DISABLE_CROSSGEN)' == '' "
           AfterTargets="PublishMSBuildExtensions">
     <ItemGroup>
-      <NETCore10Assemblies Include="$(PublishDir)/System.ComponentModel.Primitives.dll;
```

Comment:
See https://github.com/dotnet/cli/pull/6053/files#diff-acadc45977478b307ad6d53f23f6e17cR187.  You missed cleaning up usages of this Item you are removing.